pkg/models/datasource_cache_test.go

```diff
@@ -285,93 +319,23 @@ func TestDataSourceDecryptionCache(t *testing.T) {
 
 		// Populate cache
 		password, ok := ds.DecryptedValue("password")
-		So(password, ShouldEqual, "password")
-		So(ok, ShouldBeTrue)
+		require.True(t, ok)
+		assert.Equal(t, "password", password)
 
 		ds.SecureJsonData = securejsondata.GetEncryptedJsonData(map[string]string{
 			"password": "",
 		})
 		ds.Updated = time.Now()
 
 		password, ok = ds.DecryptedValue("password")
-		So(password, ShouldEqual, "")
-		So(ok, ShouldBeTrue)
+		require.True(t, ok)
+		assert.Empty(t, password)
 	})
 }
 
-func TestDataSourceSigV4Auth(t *testing.T) {
```

Comment:
Moved into `TestDataSource_GetHttpTransport` because they are `GetHttpTransport` tests. (Perhaps could keep 'em separate by distinguishing by name, like `TestDataSource_GetHttpTransport_SigV4` or something.)